src/components/MainDashboard/OpenPerscription/OpenPerscription.js

```diff
@@ -0,0 +1,86 @@
+import style from "./OpenPerscription.module.css";
+import Accordion from 'react-bootstrap/Accordion';
+
+const OpenPerscription = () => {
+    return (
+        <section className={style["section-openPerscription"]}>
+            <div className={style["div-main-content"]}>
+                <div className={style["div-header"]}>
+                    <h2 className={style["div-header-mainTitile"]}>Рецептурна Бланка</h2>
```

Comment:
There's a typo in `mainTitile` -> it should be `main-title`. In order to avoid making such mistakes, I recommend using Spell-checker extension in VS Code.

Reply:
Done